### PR TITLE
Des 12259

### PR DESCRIPTION
**Pull request by rbarros-daitan**

### CODE REVIEW

File: minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindow.cs [hunk 1]
```diff
@@ -66,15 +66,22 @@ public class ApplicationWindow : ParagonWindow, IApplicationWindowEx, IDisposabl
...
+        {
+            Activated += ApplicationWindow_Activated;
             Loaded += OnLoaded;
             var nativeWindow = new NativeApplicationWindow(this);
             nativeWindow.AddHook(WndProc);
+        }
+
+        void ApplicationWindow_Activated(object sender, EventArgs e)
+        {
+            _taskbarYellowState = false;
```
Comment (lneir) [inline minuet/Paragon.Runtime/Kernel/Windowing/ApplicationWindow.cs:84]: why is this necessary?  does flash still occur if window is focused?

**Comment by rbarros-daitan** [comment]
This is needed to disable future notification until application window is restored by user.

**Comment by lneir** [comment]
fyi, moved ticket to sprint-45.  passed deadline to make into s44. so will merge later.
